doc/source/user-guide.rst

```diff
@@ -0,0 +1,60 @@
+.. _ref_user_guide:
+
+User guide
+##########
+
+This section provides information on how to customize the Ansys Sphinx theme.
```

Comment:
```suggestion
This section provides information on how to customize the Ansys Sphinx Theme.
```

Reply:
both i wanted to give information about the basic setting such as switcher , icons and also the customization with third party extensions. so i reformatted as `This section outlines the fundamental configurations of the Ansys Sphinx theme and how to integrate 
third-party extensions with this theme to customize your documentation.`